src/api/java/openiss/Kinect.java

```diff
@@ -43,11 +49,16 @@
 import org.openkinect.freenect.VideoFormat;
 import org.openkinect.freenect.VideoHandler;
 
+import javax.imageio.ImageIO;
+
 public class Kinect {
 
 	static byte[] color;
 	static ShortBuffer depth;
 
+	private static String colorFileName = "src/api/java/openiss/ws/soap/service/color_example.jpg";
+	private  static String depthFileName = "src/api/java/openiss/ws/soap/service/depth_example.jpg";
```

Comment:
those filenames need to relative to resources directory not to the project directory. Make sure that those files exist in resources